test/web/10-VerifiableCredentialStore.js

```diff
@@ -125,11 +126,10 @@ describe('VerifiableCredentialStore', () => {
     const newCred = Object.assign({}, AlumniCredential, {id: 'foo'});
     await vcStore.insert({credential: newCred});
 
+    // query with an issuer that is not the same as the issuer on the credential
     const credentials = await vcStore.match({query: query.query1});
```

Comment:
Can we import `query` as `queries` instead -- and rename `query1` to something more semantic, i.e., something that represents what the query is for / what makes it unique from the other queries in `queries`? Maybe `queries.queryWithTrustedIssuer` or something like that?

Reply:
Updated here 99e929c